src/useKonami.ts

```diff
@@ -17,7 +18,7 @@ export default (action, {
         action();
       }
     },
-    [input, setInput, code, action],
+    [input, setInput, code, action]
```

Comment:
I prefer trailing commas, avoids these useless changes in diffs.

Reply:
Done :)